lib/Weasel/Driver/Selenium2.pm

```diff
@@ -144,7 +144,11 @@ sub start {
     do {
         if ( defined  $self->{caps}{$_}) {
             my $capability_name = $_;
-            if ( $self->{caps}{$capability_name} =~ /\$\{([^\}]+)\}/ ) {
+            if ( $self->{caps}{$capability_name} =~
+                  /\$\{             # a dollar sign nd opening brace
+                   ([^\}]+)         # any character not a closing brace
+                   \}/x             # a closing brace
+                ) {
```

Comment:
I didn't like the original code in the sense that it's lacking any useful documentation.

The new code didn't improve anything at all, I mean, yes, any regex programmer can see we want a litteral `$` and followed by a litteral opening brace.

This comment doesn't seem to comply with Chris's description of inline comments:

> comments should describe things which are outside the code -- why's and wherefore's, promises to developers, reasoning, etc. -- not the code itself

(quote from https://ledgersmb.org/content/coding-guidelines )